### PR TITLE
Vaishnavi assignment 8

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
-Vaishnavi_Assignment_8_Done

### CODE REVIEW

File: src/vaishnaviVJun20/Bank.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package vaishnaviVJun20;
+
+public class Bank {   
+	int creditCount = 0;
+	int debitCount = 0;
+	int printBalanceCount = 0;
+	static int creditCount1 = 0;
+	static int debitCount1 = 0;
+	static int printBalanceCount1 = 0;
```
Comment (KrishnaKTechnocredits) [inline src/vaishnaviVJun20/Bank.java]: Remove the initialization of all instance variables.

File: src/vaishnaviVJun20/Bank.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	int balance = 50000;
```
Comment (KrishnaKTechnocredits) [inline src/vaishnaviVJun20/Bank.java]: No Need to initialize this variable. perform credit() method first.

File: src/vaishnaviVJun20/Bank.java (hunk 1 above)
Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20/Bank.java]: Done

File: src/vaishnaviVJun20/Bank.java (hunk 2 above)
Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20/Bank.java]: Done

**Review by KrishnaKTechnocredits — COMMENTED** [review]
///

**Pull request merged**